library/VTKExtensions/Rendering/vtkF3DRenderer.cxx

```diff
@@ -1338,6 +1339,22 @@ void vtkF3DRenderer::ShowEdge(bool show)
   }
 }
 
+//----------------------------------------------------------------------------
+void vtkF3DRenderer::SetUseOrthographicProjection(bool use)
+{
+  vtkCamera* camera = GetActiveCamera();
+  if (use)
```

Comment:
how do you turn it off then ?

Reply:
Point noted, we need an else part to capture the use=FALSE case

Reply:
look at other turn on/off feature like the one just below :)

Reply:
Since this is toggle, we do not need to keep track of the state here

Reply:
also its better to use the same design as other boolean option